CoreFoundation/Base.subproj/CFUtilities.c

```diff
@@ -764,7 +767,16 @@ void CFLog(CFLogLevel lev, CFStringRef format, ...) {
 #if DEPLOYMENT_RUNTIME_SWIFT
 // Temporary as Swift cannot import varag C functions
 void CFLog1(CFLogLevel lev, CFStringRef message) {
+#ifdef __ANDROID__
+    UInt8 buffer[4096];
+    CFIndex usedBufLen = sizeof buffer-1;
+    CFStringGetBytes(message, CFRangeMake(0, CFStringGetLength(message)),
+		     kCFStringEncodingUTF8, ' ', FALSE, buffer, usedBufLen, &usedBufLen);
+    buffer[usedBufLen] = '\000';
+    __android_log_print(ANDROID_LOG_INFO, "Swift", "%s", buffer);
```

Comment:
For a tag it should probably be something other than "Swift". the process name would probably be more what users are expecting. Additionally the __android_log_print level enum can map reasonably well onto CFLogLevel.

Note: this is just window dressing imho it just makes the experience better and is not a blocking item for this PR